crates/containerd-shim-wasm/src/sandbox/containerd.rs

```diff
@@ -137,23 +431,144 @@ impl Client {
             log::info!("manifest is not in WASM OCI image format");
             return Ok((vec![], platform));
         };
+
         log::info!("found manifest with WASM OCI image format.");
+        // This label is unique across runtimes and versions
+        // a precompiled component/module will not work across different runtimes or versions
+        let label = format!(
+            "runwasi.io/precompiled/{}/{}",
```

Comment:
Could we declare this uri `runwasi.io/precompile` as a static string to be used in the entire module?

Reply:
updated